modules/netflow/configuration/logstash/netflow.conf.erb

```diff
@@ -15,6 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
+<% deprecation_logger.deprecated("The Netflow module has been deprecated and may be removed in a future release.") %>
```

Comment:
Since we are deprecating this in favor of something else, I'd recommend we do this:

 "The Netflow module has been deprecated in favor of the Beats Netflow module, and may be removed in a future release."

That said, ideally I would also like to add a link to the documentation, thoughts?